secure_message/resources/threads.py

```diff
@@ -79,9 +79,14 @@ def _validate_request(request_data, metadata):
         if not request_data:
             bound_logger.error('No properties provided')
             raise BadRequest(description="No properties provided")
+        try:
+            request_data['is_closed']
+        except KeyError:
+            bound_logger.error('Invalid properties provided')
+            raise BadRequest(description="Only 'is_closed' property may be provided")
```

Comment:
I think catching the KeyError would only make sense if it was combined with the key access in the type checking below. Seeing this makes me think it's clearer to do it this way though, but using an if:
```Python
if 'is_closed' not in request_data:
    bound_logger.error('Invalid properties provided')
    raise BadRequest(description="Only 'is_closed' property may be provided")
```

Reply:
Fixed